src/methylation/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM python:3.11-bullseye AS build
+
+ARG OUT_DIR=/tmp/ugbio
+ARG MODULE_DIR=methylation
+RUN mkdir -p $OUT_DIR
+
+COPY ./src/ ./src/
+
+# build module
+RUN python -m pip install build
+RUN python -m build --outdir $OUT_DIR --wheel ./src/$MODULE_DIR
+
+##################################################################
+FROM python:3.11-slim
+
+RUN apt-get update && apt-get install -y \
+    openjdk-17-jre-headless `# for gatk` \
+    build-essential `# for make`\
+    zlib1g-dev \
+    libbz2-dev \
+    liblzma-dev \
+    libhts-dev \
+    git \
+    wget \
+    unzip \
+    && apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && apt-get autoclean \
+
+#Install libBigWig & MethylDackel
+WORKDIR /app
+RUN git clone https://github.com/dpryan79/libBigWig.git && \
+    cd libBigWig && \
+    make && \
+    make install && \
+    cp libBigWig.a /usr/local/lib/
+
+RUN git clone https://github.com/dpryan79/MethylDackel.git && \
+    cd MethylDackel && \
+    make LIBBIGWIG="/usr/local/lib/libBigWig.a" && \
+    mkdir -p /app/bin && \
+    make install prefix=/app/bin/
+ENV PATH="/app/bin:$PATH"
+
+# Download and install gatk
+ARG GATK_VERSION=4.6.0.0
+RUN wget https://github.com/broadinstitute/gatk/releases/download/${GATK_VERSION}/gatk-${GATK_VERSION}.zip && \
+    unzip gatk-${GATK_VERSION}.zip && \
+    mv gatk-${GATK_VERSION} /opt/gatk && \
+    ln -s /opt/gatk/gatk /usr/local/bin/gatk && \
+    rm gatk-${GATK_VERSION}.zip
+
+ARG OUT_DIR=/tmp/ugbio
+
+# Install module
+COPY --from=build $OUT_DIR $OUT_DIR
+
+RUN pip install ${OUT_DIR}/*.whl
+
+COPY --from=build ./src/methylation/ugbio_methylation/reports ./src/methylation/ugbio_methylation/reports
```

Comment:
Instead of copying it in the docker file we used the pyprojec.toml:
`[tool.setuptools.package-data]
"*" = [
    "**/reports/*",
]`

Reply:
this can work if the report is running from the python code
In this case it's running directly from the wdl and therefor this COPY is needed
We did the same thing in mrd